content/blog/2020-02-10-introduction-to-machine-learning-with-the-tidyverse/index.Rmarkdown

```diff
@@ -0,0 +1,191 @@
+| 04      | [Ensembling][04-slides]                | [Exercises][04-ex], [Solutions][04-sol] |
+| 05      | [Workflows][05-slides]                 | [Exercises][05-ex], [Solutions][05-sol] |
+| 06      | [Recipes][06-slides]                   | [Exercises][06-ex], [Solutions][06-sol] |
+| 07      | [Cross-validation][07-slides]          | [Exercises][07-ex], [Solutions][07-sol] |
+| 08      | [Tuning][08-slides]                    | [Exercises][08-ex], [Solutions][08-sol] |
+
+
+## Instructor Notes
+
+We did a trial run of this workshop in December 2019 in Boston with about 20 participants, which proved one of [Greg Wilson's cardinal rules](https://teachtogether.tech/#the-rules):
+
+> "Remember that no lesson survives first contact with learners…"
+
+Actually, the workshop went pretty smooth for a first run, and we received positive feedback from our attendees. But, like any good educators, Garrett and I decided that a content renovation would make the workshop **even better**. This decision was driven by a few observations:
+
+1. We realized that the process of using a fitted model object for generating predictions was pretty new to many attendees. We needed to spend more time on this, so we beefed up our [early section on predicting][01-slides] considerably. This meant that [parsnip](https://tidymodels.github.io/parsnip/) was our first tidymodels package to introduce, which felt right! This should be your first tidymodels package to learn/teach if you are new here (previously, we had started with [rsample](https://tidymodels.github.io/rsample/)).
+
+1. Many attendees were less familiar with resampling methods in general, and in particular with [bootstrap resampling](https://moderndive.com/8-confidence-intervals.html). Since bootstrapping is such a key concept, we pushed [cross-validation][07-slides] later and added an earlier section on [sampling and resampling][03-slides].
+
+1. To lay out the red carpet for [ensembling][04-slides] (we worked up to bagging and random forest models), we spent some time working with and interpreting single decision trees, including a ["Guess the Animal" team activity](https://www.atarimagazines.com/v4n12/Animal.html) that helped to loosen everyone up on day 1.
+
+1. A new, but very much welcome, kid on the tidymodels block appeared just before our workshop in December: [workflows](https://tidymodels.github.io/workflows/). For conf, we re-factored our approach to introduce workflows by bundling together formulas and parsnip model specifications *first* (via [`add_formula()`](https://tidymodels.github.io/workflows/reference/add_formula.html) and [`add_model()`](https://tidymodels.github.io/workflows/reference/add_model.html)), *then* introducing [recipes][06-slides] as a way to move beyond formulas and do feature engineering (substiting `add_formula()` with [`add_recipe()`](https://tidymodels.github.io/workflows/reference/add_recipe.html) instead).
```

Comment:
This is exactly the kind of release notes I always want people to write about lessons - thank you.